hof.js

```diff
@@ -21,9 +21,26 @@ var people = [
 // called `peopleNames`.
 // Type your solution immediately below this line:
 
+var peopleNames = [];
```

Comment:
Because `.map` returns an array, you can assign it as the value to `var peopleNames =` and it makes the code a little cleaner

For the callback function in `people.map()`, we can give a parameter called `person` to refer to each item in that array (it can be called anything, but I like having naming the plural for the array and the singular tense of the same word in array methods).

All together, it can look something like this:

```js
var peopleNames = people.map(function(person) {
  return person.name;
});
```